package/rm2fb/package

```diff
@@ -46,11 +46,20 @@ package() {
 configure() {
     systemctl daemon-reload
     systemctl enable rm2fb --now
-    systemctl restart xochitl
+    # Restart xochitl if it's running
+    if systemctl --quiet is-active xochitl; then
```

Comment:
the exit code is 0 if xochitl is active, so should this be a negated condition?

Reply:
Oh right, forgot about that.

Reply:
Hmm, after a quick test, this actually works properly and will be truthy if systemctl is running.